multiload/data/properties.ui

```diff
@@ -0,0 +1,1424 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.38.2 -->
+<interface>
+  <requires lib="gtk+" version="3.22"/>
+  <object class="GtkImage" id="close_button_img">
+    <property name="visible">True</property>
+    <property name="can-focus">False</property>
+    <property name="icon-name">window-close</property>
+  </object>
+  <object class="GtkAdjustment" id="graph_size_spin_button_adj">
+    <property name="lower">10</property>
+    <property name="upper">1000</property>
+    <property name="value">40</property>
+    <property name="step-increment">1</property>
+    <property name="page-increment">5</property>
+  </object>
+  <object class="GtkImage" id="help_button_img">
+    <property name="visible">True</property>
+    <property name="can-focus">False</property>
+    <property name="icon-name">help-browser</property>
+  </object>
+  <object class="GtkAdjustment" id="net_threshold1_spin_button_adj">
+    <property name="lower">10</property>
+    <property name="upper">999999998</property>
+    <property name="value">1000000</property>
+    <property name="step-increment">500</property>
+    <property name="page-increment">1000</property>
+  </object>
+  <object class="GtkAdjustment" id="net_threshold2_spin_button_adj">
+    <property name="lower">11</property>
+    <property name="upper">999999999</property>
+    <property name="step-increment">500</property>
+    <property name="page-increment">1000</property>
+  </object>
+  <object class="GtkAdjustment" id="net_threshold3_spin_button_adj">
+    <property name="lower">12</property>
+    <property name="upper">1000000000</property>
+    <property name="step-increment">500</property>
+    <property name="page-increment">1000</property>
+  </object>
+  <object class="GtkAdjustment" id="speed_spin_button_adj">
+    <property name="lower">50</property>
+    <property name="upper">60000</property>
+    <property name="value">500</property>
+    <property name="step-increment">50</property>
```

Comment:
Can we use `name="step-increment">10<` please?
My favorite value seems to be 120 :)